### PR TITLE
Add categorized menu toggles to show/hide menu items

### PR DESCRIPTION
**Pull request by acchiang**
ezgif com-gif-maker

Relies on #78 which adds categories to the menu data

### CODE REVIEW

File: client/src/components/MenuSelector.js [hunk 1]
```diff
@@ -43,24 +55,55 @@ const MenuRow = ({ orderItem, updateQuantity }) => {
...
+                  {category}
+                </ToggleRow>
+              </tr>
+              {categoryItems.map((orderItem) => {
+                return (
+                  <MenuRow
+                    className={`${category}Row`}
+                    key={orderItem._id}
+                    orderItem={orderItem}
+                    // HACK
```
Comment (christylo) [inline client/src/components/MenuSelector.js:98]: lul

**Review by christylo — APPROVED** [review]
:))

File: client/src/components/MenuSelector.js (hunk 1 above)
Comment (acchiang) [reply client/src/components/MenuSelector.js:98]: lol the `// HACK` was not me lololol

Comment (acchiang) [reply client/src/components/MenuSelector.js:98]: @rzyxie was this u

**Pull request merged**